### PR TITLE
Allow nvbench to use an exiting fmt when brought in via find_package

### PR DESCRIPTION
**Pull request by robertmaynard**

### CODE REVIEW

File: cmake/NVBenchDependencies.cmake [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ################################################################################
 # fmtlib/fmt
 rapids_cpm_find(fmt 9.1.0
```
[the remark this answers is not in the document]
Comment (bdice) [reply cmake/NVBenchDependencies.cmake:3]: nvbench originally requested that feature in https://github.com/rapidsai/rapids-cmake/issues/101

**Review by allisonvacanti — APPROVED** [review]
LGTM, @robertmaynard feel free to merge after addressing @bdice's question.

**Comment by robertmaynard** [comment]
> LGTM, @robertmaynard feel free to merge after addressing @bdice's question.

Are you okay moving to rapids-cmake 23.06 to get this feature? It would mean moving your minimum CMake version to `3.23.1` compared to the existing `3.20`

**Comment by bdice** [comment]
@robertmaynard Can we finalize this PR?

**Comment by bdice** [comment]
I'd support making this update by bumping the CMake minimum. @allisonvacanti Do you have any reservations about that?

**Comment by robertmaynard** [comment]
Closing as these changes are now part of https://github.com/NVIDIA/nvbench/pull/140

**Pull request closed without merging**